packages/server/src/profile/user.entity.ts

```diff
@@ -40,4 +42,7 @@ export class UserModel extends BaseEntity {
   @OneToMany((type) => PhoneNotifModel, (notif) => notif.user)
   @Exclude()
   phoneNotifs: PhoneNotifModel[];
+
```

Comment:
Should we add Exclude here if the value is not being used on the frontend?
```suggestion

@Exclude()
```

Reply:
probably